### PR TITLE
Ensure correct app and server digests during trex-operator build

### PR DESCRIPTION
**Pull request by tkrishtop**
- Ensure correct app and server digests during trex-operator build
- Fix push for trex-container-server
- Add UTC timestamps to bash logs to track configuration/instance collisions
- Move TRex images to Python 3.9 which is the last version providing Python 2 backward compatibility

### CODE REVIEW

**Comment by tkrishtop** [comment]
check dallas ocp-4.13-vanilla example-cnf

**Comment by tkrishtop** [comment]
check dallas ocp-4.13-vanilla example-cnf

**Comment by tkrishtop** [comment]
check dallas ocp-4.13-vanilla example-cnf

**Comment by tkrishtop** [comment]
check dallas 22647 $HOME/clusterconfigs-cluster5/kubeconfig example-cnf

**Comment by tkrishtop** [comment]
check dallas ocp-4.13-vanilla example-cnf

**Comment by tkrishtop** [comment]
check dallas ocp-4.13-vanilla example-cnf

**Comment by tkrishtop** [comment]
check dallas ocp-4.13-vanilla example-cnf

**Comment by tkrishtop** [comment]
Test-Hints: force-build=true

**Comment by tkrishtop** [comment]
check dallas ocp-4.13-vanilla example-cnf

**Comment by tkrishtop** [comment]
check dallas ocp-4.13-vanilla example-cnf

**Comment by tkrishtop** [comment]
Hi @capolrik the fix we discussed today [finally worked](https://www.distributed-ci.io/jobs/60a809a8-fe13-4945-931b-8554a2730b1d/jobStates), I see my changes in the logs after ensuring the digests:

```
job-trex-app1-bz5nd.log
-- snip --
+ echo_ts 'trex-server is ready'
++ date -u +%Y-%m-%dT%H:%M:%S%z
2023-08-31T16:12:45+0000 trex-server is ready
```

File: trex-operator/Makefile [hunk 1]
```diff
@@ -67,8 +67,16 @@ deploy: kustomize
 undeploy: kustomize
 	$(KUSTOMIZE) build config/default | ${CLUSTER_CLI} delete -f -
 
+# Ensure proper images for trex-container-app and trex-container-server
```
Comment (capolrik) [inline trex-operator/Makefile:70]: Nice! It is good to see this method working. We will need at some point to change the way the T-REX operator is retrieving the digest it uses.

File: trex-container-app/Makefile [hunk 2]
```diff
@@ -41,7 +41,7 @@ build-app:
 .PHONY: push-all # Push ALL images
 push-all: push-server push-app
 
-.PHONY: push-testpmd # Push TRex Server
+.PHONY: push-server # Push TRex Server
```
Comment (capolrik) [inline trex-container-app/Makefile:44]: good catch!

**Review by capolrik — APPROVED** [review]
Let's merge this

File: trex-operator/Makefile (hunk 1 above)
Comment (tkrishtop) [reply trex-operator/Makefile:70]: 100% agree

**Pull request merged**